### PR TITLE
[Pack: Recorded Future Intelligence][v1.4.0] Recorded Future playbook alerts v1 0

### PR DESCRIPTION
**Pull request by recordedfuture-simonhornestedt**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N/A

## Description
Introduce new Recorded Future feature Playbook alerts as a separate integration into the Recorded Future content pack, some minor settings update to the Recorded Future v2 Integration.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by edik24** [comment]
Hi @recordedfuture-simonhornestedt can you please fill out the contribution Registration form and sign the CLA?

**Comment by efelmandar** [comment]
Hi @recordedfuture-simonhornestedt and thank you for your contribution, I reviewed your changes and found some points I want to discuss please see them below:

1. The new incident types you added do not use auto-extract, which is not recommended in most use cases. I suggest configuring auto-extract to all or a list to some incident fields to leverage auto-extract and auto-enrichment in playbooks.

2. In the `Recorded Future Domain Abuse` playbook - before the `Create issue in JIRA` (task no. 1) there is no check that JIRA integration is configured and available.

3. In the `Recorded Future Playbook Alert Details` playbook - Why not use the `Mapping` tab in the task to map all the output fields to the corresponding incident fields to show it better in the layout?

4. In the `Recorded Future Vulnerability` playbook, as before tasks `Create issue in JIRA` and `Enrich CVE as Note` (tasks no. 1 & no. 16) there is no check that the integration is enabled and configured.

5. The two playbooks, `Recorded Future Vulnerability` and `Recorded Future Domain Abuse` are very similar and only differ by a couple of tasks which creates a little bit of duplication and probably can be merged into one playbook.
As this might not fit every use case please consider this change or let me know if you think otherwise.

Overall the changes look good, and once these points are addressed we can move forward

**Comment by efelmandar** [comment]
@recordedfuture-simonhornestedt I appreciate your response regarding my points and can understand why it is reasonable to not make the changes in this case.  Thanks again your contributions are valued and appreciated.

**Review by efelmandar — APPROVED** [review]

File: Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+                for incident in _val:
+                    attachments = list()
+                    incident_json = json.loads(incident.get("rawJSON", "{}"))
+                    if incident_json.get("panel_evidence_summary", {}).get(
+                        "screenshots"
+                    ):
+                        for screenshot_data in incident_json["panel_evidence_summary"][
+                            "screenshots"
+                        ]:
+                            file_name = f'{screenshot_data.get("image_id").replace("img:","")}.png'
```
Comment (yucohen) [inline Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.py:178]:
```suggestion
                            file_name = f'{screenshot_data.get("image_id", "").replace("img:","")}.png'
```
In the case where no screenshots, the replace method won't fail

**Review by yucohen — CHANGES_REQUESTED** [review]
@recordedfuture-simonhornestedt Thanks for the contribution! Great work!
Please see my comments :)
Also, we can delete the pipfile and pipfile.lock files

File: Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.yml [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+configuration:
+- display: API URL (e.g., https://api.recordedfuture.com/gw/xsoar/)
+  name: server_url
+  defaultvalue: https://api.recordedfuture.com/gw/xsoar/
+  type: 0
+  required: true
+- display: API Token
+  name: token
+  defaultvalue: ""
+  type: 4
```
[the remark this answers is not in the document]
Comment (recordedfuture-simonhornestedt) [reply Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.yml]: done

File: Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.py (hunk 1 above)
Comment (recordedfuture-simonhornestedt) [reply Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.py]: done

File: Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.py [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+                            file_data = screenshot_data.get("base64")
```
[the remark this answers is not in the document]
Comment (recordedfuture-simonhornestedt) [reply Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.py]: done

**Comment by recordedfuture-simonhornestedt** [comment]
Removed pipfile and updated the dockerimage tag so that should be the last of the changes requested

**Review by yucohen — APPROVED** [review]

**Pull request merged**